src/framework/lib/genesisframework/task.rb

```diff
@@ -124,7 +124,9 @@ def install provider, *what
                 || Gem::Dependency.new(item).matching_specs.count == 0
             end
             if gems.size > 0    # make sure we still have something to do
-              Kernel.system('gem', 'install', '--no-ri', '--no-rdoc', *gems)
+              options = ['--no-ri', '--no-rdoc', config.fetch(:gems_source, '').split]
```

Comment:
In that case, can we name the key something like `:gem_args` or `:gem_flags`, since this could potentially be used to add other flags to the gem install command.